src/vs/platform/windows/common/windows.ts

```diff
@@ -19,6 +19,7 @@ export const WindowMinimumSize = {
 
 export interface IBaseOpenWindowsOptions {
 	readonly forceReuseWindow?: boolean;
+	readonly remoteAuthority?: string | null;  /* used when no inputs are provided or inputs are neither file nor vscode-remote. Null forces a local window */
```

Comment:
```suggestion
		/**
	 * Only used when a window is opened with either
	 * no workspace or with a workspace that is
	 * neither `file://` nor `vscode-remote://`
	 * protocol to indicate the remote authority
	 * that should be used.
	 *
	 * Use `null` to explicitly force a window without
	 * remote authority.
	 */
	readonly remoteAuthority?: string | null;
```

Reply:
@aeschli comments like yours do not appear in our hovers, can you please makes sure to use JSDoc compliant comments for APIs, even if they are internal? I think otherwise people will not understand what the option is about since it has very specific behavour.